### ISSUE TITLE
Sort containers

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit-podman#1097]
The containers list should be sortable. It should default to the container's name, but you should be able to sort by owner, CPU and memory usage, and state.

This can be done by making the headers clickable.

- composable: https://www.patternfly.org/v4/components/table#composable-sortable---custom-control
- legacy: https://www.patternfly.org/v4/components/table/react-legacy#sortable---custom-control

**Comment by jelly** [comment]
For the tables we use a `ListingPanel` from Cockpit which is our own custom Table implementation so I think we would have to switch to a TableComposable from what I'm seeing.

**Comment by garrett** [comment]
We'd either want to switch to a `TableComposable` directly in Podman or switch `ListingPanel` over to one.

**Comment by jelly** [comment]
We can actually already add sorting, I was wrong. But with pods it get's a bit akward:

![image](https://user-images.githubusercontent.com/67428/225905071-1bc5409a-57f2-40cb-aafa-983d4fcc4f55.png)

**Comment by jelly** [comment]
Nevermind, we always showed pods on the bottom, so this is actually fine and working as intended.

**Comment by jelly** [comment]
@garrett is the current behavior acceptable or does it still need to be enhanced?

**Comment by garrett** [comment]
Yeah, keep pods in groups. I see the pods also have headers; you can sort the containers within a pod too?

I don't think mixing podless containers with those from pods is a good idea, as when you're doing container management, you either have standalone containers you act upon individually or treat the whole pod together as one (and not the containers within pods) — at least generally. In other words, you would usually treat pods as a combined unit (start, stop, delete).

**Comment by cmd430** [comment]
would it be possible add tabs to the list so we have two lists? 
a tab that lists all pods (and their containers) and a tab that just lists containers not in pods
with a lot of pods / containers it would be nice if they were separated

**Comment by garrett** [comment]
Seeing only containers outside of pods and containers in pods should probably go into filtering, which we should improve anyway, instead of adding a tab based UI.
